src/components/views/right_panel/UserInfo.tsx

```diff
@@ -934,17 +929,14 @@ function useRoomPermissions(cli: MatrixClient, room: Room, user: RoomMember): IR
 
         let modifyLevelMax = -1;
         if (canAffectUser) {
-            const editPowerLevel = (
-                (powerLevels.events ? powerLevels.events["m.room.power_levels"] : null) ||
-                powerLevels.state_default
-            );
-            if (me.powerLevel >= editPowerLevel && (isMe || me.powerLevel > them.powerLevel)) {
```

Comment:
I think we still need the `isMe || me > them` check to ensure the UI doesn't enable itself when looking at a fellow Admin?

Reply:
The check is done as part of `canAffectUser` which we're inside an if for

Reply:
so it is